### DISCUSSION TITLE
issue: Control show the previous val in render

### DISCUSSION THREAD

**Discussion by YangChen00** [react-hook-form/react-hook-form#7281] [Q&A]
### Version Number

7.0.5

### Codesandbox/Expo snack

https://codesandbox.io/s/calc-forked-xu61t?file=/src/App.tsx

### Steps to reproduce

1. open the codesandbox: https://codesandbox.io/s/calc-forked-xu61t?file=/src/App.tsx
2. Delete the default value row
3. Hit Add
4. You can see the {quantity} show the previous control's value

Notice: I change the append value, not set the new quality in new row

### Expected behaviour

After I hit the add button, show blank in the {quantity} field

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
https://react-hook-form.com/api/usefieldarray

> When you append, prepend, insert and update the field array, the obj can't be empty object rather need to supply all your input's defaultValues.

**Reply by YangChen00** [reply]
Yes, that's how I'm currently practicing it, not sure if it's designed for performance optimization considerations
